### ISSUE TITLE
A18_comma

### ISSUE THREAD

**Issue by ToddlerPichu** [NaomiMod/Rent-A-Hero-No1#101]
The comma breaks to the second line causing the whole sentence to spill over, perhaps a fix of:
```
Even after he beat Segata Sanshiro,
he still seems strange.
He is still pumped up on something.
```
![shot326](https://user-images.githubusercontent.com/41944249/193424934-684cb042-af52-4a52-9aaa-ca27c8b2841d.png)

**Comment by VincentNLOBJ** [comment]
Fixed.
Please suspend your testing for a few mins, I'll send you new patch asap!

**Comment by ToddlerPichu** [comment]
I see you fixed and mentioned in #100 as I was typing, I apologies for the reports it seems you were fast it... I will suspend testing until the new patch thank you 👍
